docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md

```diff
@@ -34,6 +28,12 @@ For example: No courses selected then the default flow is taken.
 
 ![An inclusive gateway has decided to take the step to prepare salad as the default because none of the conditions were fulfilled.](assets/inclusive-gateway-default.png)
 
+A converging exclusive gateway, also known as a merging or joining exclusive gateway, waits until an all executed sequence flows have arrived, then merges the separate flows and continues the process. To use a converging inclusive gateway, there must be an associated diverging gateway earlier within the process flow. This allows the converging gateway to wait for the active paths and their conditions.
```

Comment:
:x: We need to rephrase the description.

> A converging exclusive gateway, also known as a merging or joining exclusive gateway

This is a converging **inclusive** gateway. :wink: 

> waits until an all executed sequence flows have arrived, then merges the separate flows and continues the process.

This is not precise enough. 

A converging gateway is completed (i.e. merges the flows) if one of the following conditions is fulfilled:
- if all incoming sequence flows were taken at least once 
- if no path exists from any active flow node instance to the inclusive gateway, excluding paths that end on an already taken incoming sequence flow of the gateway

> To use a converging inclusive gateway, there must be an associated diverging gateway earlier within the process flow. This allows the converging gateway to wait for the active paths and their conditions.

This is not true. Usually, a converging gateway is used together with a diverging gateway. However, this is not a requirement. A converging inclusive gateway could be used with other elements.

Let's remove this part.

Reply:
I've tried to clean this up to remove some jargon while still being specific: 

A converging inclusive gateway (also known as a merging or joining exclusive gateway) merges incoming paths before the sequence flow continues. A converging gateway is completed and merges incoming sequence flows if one of the following conditions is met:
- All incoming sequence flows have been taken at least once.
- No path exists from any active flow node to the inclusive gateway (excluding incoming paths to the inclusive gateway that have already been taken).

Let me know if this still captures the specificity required.